caliban/docker/build.py

```diff
@@ -248,8 +256,28 @@ def _dependency_entries(workdir: str,
   return ret
 
 
-def _package_entries(workdir: str, user_id: int, user_group: int,
-                     package: u.Package) -> str:
+def _wrapper_script_path() -> str:
+  return resource_filename('caliban.resources', WRAPPER_SCRIPT)
+
+
+def _cloud_sql_proxy_wrapper_path() -> str:
+  return resource_filename('caliban.resources', CLOUD_SQL_WRAPPER_SCRIPT)
+
+
+def _mlflow_dockerfile(mlflow_cfg: Dict[str, Any]) -> str:
```

Comment:
name isn't quite right, you're not returning a Dockerfile. Also this isn't mlflow, this is about `cloud_sql_proxy`

Reply:
of course, will change